stable/kibana/templates/deployment.yaml

```diff
@@ -106,10 +106,12 @@ spec:
         - name: "{{ $key }}"
           value: "{{ $value }}"
         {{- end }}
+{{- if .Values.service.disableInternalPort }}
```

Comment:
As per my comment, it seems like the intended behaviour would warrant the following instead:
```yaml
{{- if not .Values.service.disableInternalPort }}
```

Reply:
@yurrriq: I'll send another PR to fix this.

Reply:
This is fixed in https://github.com/helm/charts/pull/10932